data/tribes/buildings/warehouses/barbarians/warehouse/init.lua

```diff
@@ -1,8 +1,13 @@
 dirname = path.dirname(__file__)
 
-animations = {}
-add_animation(animations, "idle", dirname, "idle", { 60, 78 })
-add_animation(animations, "build", dirname, "build", { 60, 78 })
+animations = {
```

Comment:
no need to have local variables here, move this table down into the unit definition (for all units where this table is used in only one place)